src/Angor/Client/Pages/Create.razor

```diff
@@ -1199,6 +1201,44 @@
         }
     }
 
+    private async Task RecalculateFee(decimal customFeeRate)
```

Comment:
The method `FeeRangeChanged` should call this method otherwise we have duplicated code